src/main/java/com/epam/tc/hw3/library/utils/GetProperties.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.tc.hw3.library.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+
+public class GetProperties {
+
+
+    private String username;
+    private String password;
```

Comment:
What would you do if you have 2 users?

Reply:
Forget about it. If you have more than 2 users, you just wouldn't use property file like this

Reply:
I fixed it anyway